docs/tutorial.md

```diff
@@ -100,22 +107,44 @@ juju integrate discourse-k8s nginx-ingress-integrator
 ### Create an admin user and log in
 
 To create an admin user, use the `create-user` action:
-```
+```bash
 juju run discourse-k8s/0 create-user admin=true email=email@example.com
 ```
-The command will return the password of the created user. Discourse will be deployed with `discourse-k8s` as default hostname. In order to reach it, modify your `/etc/hosts` file so that it points to `127.0.0.1`:
+The command will return the password of the created user. Discourse will be deployed with `discourse-k8s` as default hostname.
+If you are using a Multipass instance you need to forward the request from your local to the Multipass instance.
+First get the Multipass instances IP address. Since the indico is served on the local address of the Multipass VM we need to use the ip address of the VM. To get the IP address of a Multipass instance run the following command:
+
+```bash
+ip a | awk '/inet .* ens3/{print $2}' | cut -d'/' -f1
```

Comment:
This will fail in many cases (for example if not using the qemu backend and instead using the lxd backend in multipass). I would use instead:
`ip -4 -j route get 2.2.2.2 | jq -r '.[] | .prefsrc'`

Reply:
Thank you!!!!